src/data-tutorials/users.tsx

```diff
@@ -2727,6 +2727,13 @@ const Users: User[] = [
     tags: ['celosage', 'intermediate', 'smartcontract', 'solidity'], 
 
   },
+  {
+    title: 'Building a Simple DeFi Application on the Celo Blockchain Using Python',
+    description: 'In this tutorial, we will build a simple decentralized finance (DeFi) application on the Celo blockchain using Python',
+    preview: require('./showcase/intermediate/building-a-simple-defi-application-on-the-celo-blockchain-using-python.png'),
+    website: 'blog/tutorials/building-a-simple-defi-application-on-the-celo-blockchain-using-python',
```

Comment:
Can you edit the name of your header image file here,  It ha to be the exact same as the image name you added in the intermediate folder... hence the cause of it's undeployment